Experimental/conda/readme.md

```diff
@@ -0,0 +1,37 @@
+Create a conda environment for running fab
+```
+conda env create -f environment.yml
+```
+
+
+Activate the new environment
+
+```
+conda activate fab
```

Comment:
This doesn't match the `sci-fab` value in the environment.yml file.

Reply:
Thanks!